dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
shouldn't this file be at dockerfiles/tests ?
as base is the base image, and other images extending base are in dockerfiles/directory